slides/intro-02-data-budget.qmd

```diff
@@ -42,17 +42,18 @@ knitr:
 :::
 
 ::: footer
-Credit: <https://www.svgrepo.com/svg/8322/taxi>
+Credit: <https://www.svgrepo.com/svg/8322/forested>
 :::
 
-## Data on Chicago taxi trips
+## Data on forests in Washington
 
 ::: columns
 ::: {.column width="60%"}
--   `N = 10,000`
--   A nominal outcome, `tip`, with levels `"yes"` and `"no"`
--   Several **nominal** variables like pickup & dropoff location, taxi ID, and payment type.
--   Several **numeric** variables like trip length and fare subtotals.
+-   `N = 7,109` plots of land, one from each of 7,109 6000-acre hexagons in WA.
+-   A nominal outcome, `forested`, with levels `"Yes"` and `"No"`, measured "on-the-ground."
+-   18 remotely-sensed and easily-accessible predictors:
+     - **numeric** variables based on weather and topography.
+     - **nominal** variables based on classifications from other governmental orgs.
```

Comment:
I know that we otherwise leave EDA for the end of this deck, but for Early Value's sake, should we include a nice plot of Washington with `forested` Yes/No here?